IPMUinternal/PFS-IPMU-00001/startdown.rst

```diff
@@ -17,120 +26,159 @@ Start backends first, end point to external, and monitoring systems.
 
-* Start backup service (``backup``)
+* Start backup service
 
-  * *backup*
+  * *backup* : ``backup`` on **dlb5-vm**
 
-* Start external servers (``pfs``, ``pfspipe``)
+* Start external servers
 
-  * *pfssrv*, *pfspipe2*
+  * *pfs* : ``pfssrv`` on **dlc2-vm**
+  * *pfspipe* : ``pfspipe2`` on **dlc2-vm**
 
 * Start service VM guests
 
-  * ``jirapipe`` : *jira-pipe-7.7*
+  * *jirapipe* : ``jira-pipe-8.3`` on **dlb3-vm**
 
     * Start jira server instance manually via systemd
 
-	ssh pfs@jirapipe-77
+	ssh pfs@jirapipe-83
 
 	sudo systemctl start jira
 
-  * *jupyter-spt*, *jupyter1*
+  check on browser whether JIRA is starting up
+
+  * *jupyter-spt* : ``jupyter-spt`` on **dlc3-vm**
+  * *jupyter1* : ``jupyter1`` on **dlb4-vm**
+  * *dbsim2* : ``dbsim2-spt`` on **rcc1-vm**
 
-* Start monitoring services (``prometheus``, ``influxdb``)
+* Start monitoring services
 
-  * *prometheus*, *influxdb*, *prometheus-snmp*
+  * *prometheus* : ``prometheus`` on **dlb5-vm**
+  * *influxdb* : ``influxdb`` on **dlb3-vm**
+  * *prometheus-snmp* : ``prometheus-snmp`` on **dlb5-vm**
 
 * Start internal services
 
-  * *landfill*, *stretch*, *jessie-main*
-  * ``Windows`` : *WinP1*, *WinVSdev* and CAD
+  * ``landfill``, ``stretch``, and ``jessie-main`` on **dlb3-vm**
+  * *Windows* : ``WinP1``, ``WinVSdev`` (and CAD?) on **dlb4-vm**
 
 * Check the above VM guests running
 * Check internal services up by accessing them.
 
 Full shutdown
 ======
 
-Shutdown end point to external to prevent unwanted connection first, and 
-shutdown backends. 
-See the section `Start & shutdown VM guest with virsh`_ for the virsh command to shutdown the VM guests.
-Refer the mail (subject: \"List of virt VMs\") sent to admin weekly for VM guests running on individual hosts.
+Shutdown the end-point to the external services to prevent unwanted connection first, and 
+then back-end services. 
+See `Start & shutdown VM guest with virsh` section for the `virsh` command to shutdown the VM guests.
+Refer the mail (subject: \"List of virt VMs\") sent to admin weekly for VM guests running on individual hosts just in case.
 
-* Shutdown monitoring services (e.g. ``prometheus``, ``influxdb``) not to get unwanted (or obvious) warnings from hosts shut down.
+* Shutdown monitoring services (e.g. *prometheus*, *influxdb*) not to get unwanted (or obvious) warnings from hosts shut down.
 
-  * *prometheus*, *influxdb*, *prometheus-snmp*
+  * *prometheus* : ``prometheus`` on **dlb5-vm**
+  * *influxdb* : ``influxdb`` on **dlb3-vm**
+  * *prometheus-snmp* : ``prometheus-snmp`` on **dlb5-vm**
 
-* Shutdown internal servicse (like working shell host, simulator)
+* Shutdown internal services (like working shell host, simulator)
 
-  * *lf??*, *landfill*, *nfs??*, *stretch*, *jessie-main*
+  * ``jessie-main``, ``landfill``, and ``stretch`` on **dlb3-vm**
+  * ``nfs??`` on **dlb2-vm** (disused)
```

Comment:
What do you mean by "disused"?
Maybe you are trying to say that hey are not always used?

Reply:
I think we no longer use `nfs??` and `lf??`, so now we can remove these two, right?

Reply:
Yes for `nfs??`, although `nfs11` exists on **dlb2-vm**. No for `lf??` actually I'm thinking to use one of them for troubleshooting about full-memory things.